src/main.c

```diff
@@ -326,7 +326,12 @@ static uint32_t softdev_init(bool init_softdevice)
       .accuracy     = NRF_CLOCK_LF_ACCURACY_250_PPM
   };
 
+  #ifndef ANT_LICENSE_KEY
```

Comment:
maybe we should depend on USE_S340 here so we don't call the wrong function? also minor nit:no need to negate, best to flip cases for readibility.

Reply:
Flipping is no problem. The condition is better for ANT_LICENSE_KEY, IMHO, since the new code actively uses ANT_LICENSE_KEY. Switching to USE_340 would solve one problem but would create a similar one: the risk of calling the method with undefined symbol. Moreover, this a C-linkage function, passing an extra parameter (ANT_LICENSE_KEY) to the S140 variant of the method would make no harm: the extra  arg. would just silently sit on the heap, without any harm.

Reply:
Actually  the code ".source       = NRF_CLOCK_LF_SRC_RC", that is, the usage of the RC oscillator instead of the quartz is the big-big problem here, since the low-accuracy RC oscillator won't fly for ANT :/

Reply:
Commit e3683b3 makes the requested flip